qCC/ccPointPairRegistrationDlg.cpp

```diff
@@ -1333,9 +1350,22 @@ bool ccPointPairRegistrationDlg::callHornRegistration(	CCCoreLib::PointProjectio
 					QTableWidgetItem* itemA = new QTableWidgetItem();
 					itemA->setData(Qt::EditRole, dist);
 					alignedPointsTableWidget->setItem(i, RMS_COL_INDEX, itemA);
+
 					QTableWidgetItem* itemR = new QTableWidgetItem();
 					itemR->setData(Qt::EditRole, dist);
 					refPointsTableWidget->setItem(i, RMS_COL_INDEX, itemR);
+
+					//update errors along Axis
+					for (unsigned j = 0; j < 3; ++j) {
+						double diffAlongAxis = Lit[j] - Ri->toDouble()[j];
```

Comment:
```suggestion
					for (unsigned j = 0; j < 3; ++j)
					{
						double diffAlongAxis = Lit[j] - Ri->toDouble()[j];
```

Reply:
Maybe it would be good to add above a line like:
`const CCVector3d Rid = Ri->toDouble();`

And then use Rid here and there to avoid too many conversions.